prisma/schema.prisma

```diff
@@ -1,13 +1,13 @@
-generator client {
-  provider = "prisma-client-js"
-}
-
 datasource db {
   provider          = "postgresql"
   url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
 
+generator client {
+  provider = "prisma-client-js"
+}
+
 model competition {
```

Comment:
You might want to change the model to start with an uppercase to fit in with the prisma standards